sql-to-dbsp-compiler/lib/sqllib/src/array.rs

```diff
@@ -613,3 +613,38 @@ where
         Some(value) => map_index_N_(value, map_index),
     }
 }
+
+/////////////// Variant index
+
+// Return type is always Option<Variant>, but result is never None, always a Variant
```

Comment:
you don't want them to be static methods in a impl block for variant?

```
impl Variant {
fn indexV_N(self: Self, index: Option<T>) -> {
blabla}
}

bla.indexV_N(42);
```

Reply:
so far the compiler always calls functions, sometimes the functions call methods